project/AppDependencies.scala

```diff
@@ -19,7 +21,20 @@ object AppDependencies {
     "org.typelevel"                         %% "cats-core"                    % "2.6.1",
     "org.commonjava.googlecode.markdown4j"  %  "markdown4j"                   % "2.2-cj-1.1",
     "com.typesafe.play"                     %% "play-json"                    % playJsonVersion,
-    "com.typesafe.play"                     %% "play-json-joda"               % playJsonVersion
+    "com.typesafe.play"                     %% "play-json-joda"               % playJsonVersion,
+    "io.swagger.parser.v3"                  %  "swagger-parser"               % "2.1.9"
+      excludeAll(
+        ExclusionRule("com.fasterxml.jackson.core", "jackson-databind"),
+        ExclusionRule("com.fasterxml.jackson.core", "jackson-core"),
+        ExclusionRule("com.fasterxml.jackson.core", "jackson-annotations"),
+        ExclusionRule("com.fasterxml.jackson.dataformat", "jackson-dataformat-yaml"),
+        ExclusionRule("com.fasterxml.jackson.datatype", "jackson-datatype-jsr310")
+      ),
+    "com.fasterxml.jackson.core" % "jackson-core" % jacksonVersion,
```

Comment:
Would  be nice to format these in columns, like the lines above.

Reply:
I'll format the whole codebase again